README.md

```diff
@@ -3510,6 +3511,22 @@ device and then follow appropriate instructions. The Jellyfish knowledgebase
 includes an entry for doing this (search for 'Recovery data from a dead
 device').
 
+### 12. Device connectivity status
+
+When the device's network connectivity is fully operational the status displays as Online. However, these two conditions which make up this value may run into issues and result in Heartbeat Only or VPN Only.
+
+#### 12.1 Heartbeat Only
+
+When a device is in this state this indicates that the device has internet connectivity and can poll the cloud for new updates to apply. However, if you try to perform any actions which do not change the target state or want access via SSH then you will encounter network issues because the device is only accessible from the VPN. Purging data, restarting/shutdown, etc, actions require the VPN because these requests are proxied from the dashboard API to the [Supervisor API](https://www.balena.io/docs/reference/supervisor/supervisor-api/) on device and so will fail with network issues as well.
```

Comment:
```suggestion
A device with a Heartbeat Only status has internet connectivity and can poll the cloud for new updates to apply. However, performing any actions which do not change the target state or attempting to access the device via SSH will error, as these actions only have effect with a VPN connection. Actions that do not change the target state include purging data, restarting, rebooting, or shutting down, and they all require the VPN because they are proxied from the dashboard API to the [Supervisor API](https://www.balena.io/docs/reference/supervisor/supervisor-api/) on device.
```